### ISSUE TITLE
result of the preprocessor is not testable

### ISSUE THREAD

**Issue by bobagold** [ikr/couch-js-devkit#3]
Result of work of the preprocessor sometimes can be incorrect.
Maybe it should work in more automatic way, resolve require()s and/or warn user if there are unresolved require()s.

**Comment by ikr** [comment]
Yes, probably it could do a better job. I'm not doing anything with CouchDB on the current project, so I can't really tell when I'll be able to follow up on that. But pull requests are very welcome, of course.
